reference/features/search_parameters.md

```diff
@@ -364,6 +362,8 @@ If you set `matches` to `true` and search for `winter feast`:
 
 You would get the following response with **information about the matches in the `_matchesInfo` object**:
 
+Note how MeiliSearch searches for `winter` and `feast` separately because of the whitespace.
+
```

Comment:
Sorry, I'm being nitpicky, but the it felt weird to end one sentence with `:` and the immediate thing following it not being an example.

```suggestion
You would get the following response with **information about the matches in the `_matchesInfo` object**. Note how MeiliSearch searches for `winter` and `feast` separately because of the whitespace:

```